.github/ISSUE_TEMPLATE/proposal_template.md

```diff
@@ -1,10 +1,12 @@
 ---
-name: Language Proposal
-about: Fully specified language proposal
+name: Create a language specification
+about: For proposals that already have support in community discussions or from a team member.
 title: "[Proposal]: [FEATURE_NAME]"
 ---
 <!--
-New language feature proposals should fully fill out this template. This should include a complete detailed design, which describes the syntax of the feature, what that syntax means, and how it affects current parts of the spec. Please make sure to point out specific spec sections that need to be updated for this feature. If you do not have the knowledge or experience to fill this out completely, that's perfectly alright: please open this proposal as a Discussion instead (https://github.com/dotnet/csharplang/discussions/new) and the community can generally discuss the proposal in less formal terms.
+Hello, and thanks for your interest in contributing to C#! If this is your first time opening an issue, please open a discussion marked [draft issue] at https://github.com/dotnet/csharplang/discussions/new and we'll try to give you feedback on how to get to an issue-ready proposal. Without existing community discussions or invitation from a team member, we will likely convert this issue into a discussion.
+
+New language feature proposals should fully fill out this template. This should include a complete detailed design, which describes the syntax of the feature, what that syntax means, and how it affects current parts of the spec. Please make sure to point out specific spec sections that need to be updated for this feature.
```

Comment:
>Please make sure to point out specific spec sections that need to be updated for this feature.

Isn't that too much? Personally I don't remember doing that for a few of my proposals that eventually got implemented.

I agree with HaloFour's https://github.com/dotnet/csharplang/discussions/3878#discussioncomment-77043. That shouldn't be required for a community proposal.

Reply:
I don't want to change this text :). Yes, there are some proposals that do not have this, but we should be encouraging it for future proposals, not removing it.

Reply:
Works for me. @alrz Since this pull request is not adding anything new along these lines, could you use a new discussion to raise any concerns you still have?

Reply:
I don't have anything to add to the discussion I linked to. I don't believe that helps in communicating the feature. 

>A "proposal" should clearly identify a problem and a way through which that problem might be solved. That could include proposed syntax with an explanation as to what that syntax does and how it works with existing syntax. But something like a specification fragment I think is way too far. That is what the LDG should be doing, and will be doing. Any proposal that gets that far into the details is likely to pigeonhole itself and either be unusable or get shredded to pieces regardless of how much community assistance is provided. And I think without some indication of interest by the LDG there's little reason for the community to spend this time and effort to come up with something formal. You guys could spend months coming up with all of the exact specifications to support optional semicolons and it'll likely still be dead-on-arrival.

Reply:
> And I think without some indication of interest by the LDG there's little reason for the community to spend this time and effort to come up with something formal.

The point is to communicate exactly that sentiment. Issues should pretty much always start as discussions.

Reply:
If so def consider adding "Open issues at the request of a team member". IIRC we had that note somewhere in the past?

Reply:
I'd be fine with such a note. @jnm2, do you want to add it?

Reply:
Sure! We have a less strong version of that note already which is what this PR is adding, so I'm not sure what you're picturing. Replacing everything I'm adding and everything that was already there with this?

> Hello, and thanks for your interest in contributing to C#! If you haven't been invited by a team member to open an issue, please instead open a discussion marked [draft issue] at https://github.com/dotnet/csharplang/discussions/new and we'll try to give you feedback on how to get to an issue-ready proposal.

Reply:
That seems reasonable to me.

Reply:
Rereading, I'm not sure if the consensus was to leave or remove this text which exists prior to this PR:

> New language feature proposals should fully fill out this template. This should include a complete detailed design, which describes the syntax of the feature, what that syntax means, and how it affects current parts of the spec. Please make sure to point out specific spec sections that need to be updated for this feature.

Were you concluding that this text is good so long as the initial text says not to open issues without invitation? Or that this text should be replaced by text that says not to open issues without invitation?

Reply:
That stays, note added.

Reply:
Done.

Reply:
I also updated the template tag line to be consistent with this change. Wording suggestions welcome.